PubSub/src/BatchPublisher.php

```diff
@@ -64,11 +64,28 @@ class BatchPublisher
      */
     private $client;
 
+    /**
+     * @var bool
+     */
+    private $enableCompression;
+
+    /**
+     * @var int
+     */
+    private $compressionBytesThreshold;
```

Comment:
We can go ahead and add typehints to new properties!
```suggestion
    private bool $compressionBytesThreshold;
```

Reply:
Ohh, great.🙌

I've been waiting for this as this feels more natural.